pkg/server/ca/manager/slot.go

```diff
@@ -347,6 +347,10 @@ func (s *SlotLoader) loadX509CASlotFromEntry(ctx context.Context, entry *journal
 		return nil, "no slot id", nil
 	}
 
+	if entry.GetNotAfter() < time.Now().Unix() {
```

Comment:
nit: no need to use the getter.
```suggestion
	if entry.NotAfter < time.Now().Unix() {
```